Documentation/deploy-master.md

```diff
@@ -1,11 +1,13 @@
 ## Deploy Kubernetes Master Machine
 
-Boot a single CoreOS machine which will be used as the Kubernetes master.
+Boot a CoreOS machine which will be used as the Kubernetes master.
```

Comment:
...or just make it clear that we are starting with one, and more can be added later